src/events.js

```diff
@@ -58,12 +58,18 @@ Object.keys(eventMap).forEach(key => {
       })
     }
 
-    // Approximate dataTransfer on the event object
-    // jsdom does not support DataTransfer constructor
-    // https://github.com/testing-library/react-testing-library/issues/339#issuecomment-526310225
     const {dataTransfer} = eventInit
     if (typeof dataTransfer === 'object') {
-      Object.assign(event, {dataTransfer})
+      /* istanbul ignore if  */
```

Comment:
Let's add a comment explaining why we're doing this.

```suggestion
      // DataTransfer is not supported in jsdom: https://github.com/jsdom/jsdom/issues/1568
      /* istanbul ignore if  */
```